back/engines/commercial/project_management/spec/models/email_campaigns/campaigns/moderator_digest_spec.rb

```diff
@@ -53,6 +53,13 @@
         command.dig(:event_payload, :top_ideas, 0, :author_name)
       ).to eq(expected_author_name)
     end
+
+    # added to reproduce the bug and test the fix
+    it 'handles gracefully absent projects' do
```

Comment:
```suggestion
    it 'gracefully handles absent projects' do
```
(Pedantic grammar improvement)